varia/sass/elements/_config.scss

```diff
@@ -15,7 +15,7 @@ $config-elements: (
 				"fallback": map-deep-get($config-global, "font", "family", "secondary", "fallback"),
 				"css-var": map-deep-get($config-global, "font", "family", "secondary", "css-var"),
 			),
-			"line-height": map-deep-get($config-global, "font", "line-height", "md"),
```

Comment:
`$config-global` doesn't have this value, so I've changed it to `body` instead. Is that ok in this instance?